### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by Comandeer**

### CODE REVIEW

**Review by KarolDawidziuk — CHANGES_REQUESTED** [review]
Hi @Comandeer,

There are a few packages that we can bump, so please see my inline comments.
Also, as the `tslint` is deprecated we should check if there is a chance to use another one or discuss what we can do with it.

File: package.json [hunk 1]
```diff
@@ -30,45 +30,45 @@
...
 		"jasmine-spec-reporter": "^7.0.0",
-		"karma": "^6.3.16",
+		"karma": "^6.4.1",
 		"karma-browserstack-launcher": "^1.6.0",
 		"karma-chrome-launcher": "~3.1.0",
 		"karma-coverage-istanbul-reporter": "~3.0.3",
 		"karma-firefox-launcher": "^2.1.2",
-		"karma-jasmine": "^4.0.1",
+		"karma-jasmine": "^4.0.2",
 		"karma-jasmine-html-reporter": "^1.7.0",
```
[the remark this answers is not in the document]
Comment (Comandeer) [reply package.json:61]: I've updated `jasmine-core` to v4.

**Comment by Comandeer** [comment]
@KarolDawidziuk, thx for the review! I've updated the packages you mentioned, excluding `zone-js` (the version of Angular we're using requires `~0.11.4`). I've also switched CI from local Chrome to BrowserStack one and it seems that tests are passing again :tada: 

I haven't also made a switch from TSLint to ESLint as it requires changes in configuration (e.g. removing `tsconfig.base.json`).

**Review by KarolDawidziuk — APPROVED** [review]
LGTM! 🎉

**Pull request merged**